server.go

```diff
@@ -0,0 +1,104 @@
+package auditexporter
+
+import (
+	"bufio"
+	"encoding/json"
+	"net"
+
+	"github.com/hashicorp/vault/audit"
+	log "github.com/sirupsen/logrus"
+)
+
+// Listen listens on the network and address specified and sends entries to
+// the AuditHandler instance.
```

Comment:
This comment is potentially misleading. The listener is generic, it's `AcceptConnections` that sets up the handler, etc.

Maybe some of the setup logic in `main()` needs to be moved to a function here that takes a queue as well as the `Listen()` args, calls `AcceptConnections()`, and returns either the listener (with the cleanup logic in `main()` still) or a cleanup closure (for `main()` to `defer` as-is)?

Reply:
Meh, I just noticed that `AcceptConnections()` loops forever around `listener.Accept()`. Maybe the function above should be a self-contained `RunServer()` that does its own cleanup or whatever.